src/main/java/uk/gov/hmcts/reform/prl/constants/PrlAppsConstants.java

```diff
@@ -340,5 +340,9 @@ public class PrlAppsConstants {
     public static final String CASE_NAME_HMCTS_INTERNAL = "caseNameHmctsInternal";
     //C100 default court details
     public static final String C100_DEFAULT_COURT_NAME = "STOKE ON TRENT TRIBUNAL HEARING CENTRE";
+    public static final String CLOSED_STATE = State.ALL_FINAL_ORDERS_ISSUED.getValue();
```

Comment:
Can we use state.java please for this? (line 343)

Reply:
Hi @AlokDatta, It's using State.java but these have defined as constants with meaningful state names.